File: 0001 - Two Sum/PHP/Solution.php
<?php

class Solution {

    /**
     * @param Int[] $nums
     * @param Int $target
     * @return Int[]
     */
    function twoSum($nums, $target) {
       for($i = 0; $i < count($nums); $i++)  {
          $num = $nums[$i];
          $diff = $target - $num;
              for($j = $i+1; $j < count($nums); $j++) {
                  if($nums[$j] == $diff) {
                      $res = [$i,$j];
                      break;
                  }
             }
		}
        
        return $res ?? [];
    }
}

File: 0001 - Two Sum/Kotlin/Solution.kt
class Solution {
    fun twoSum(nums: IntArray, target: Int): IntArray {
        val map = HashMap<Int, Int>()
        for (i in nums.indices) {
            if (map.containsKey(target - nums[i])) {
                return intArrayOf(map[target - nums[i]]!!, i)
            }
            map[nums[i]] = i
        }
        return intArrayOf(-1, -1)
    }
}